<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsPage_Editor</class>
 <widget class="QWidget" name="SettingsPage_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings - Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBoxGeneral">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="checkBoxLimitWidth">
          <property name="text">
           <string>Maximum line length</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxTextWidth">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>5000</number>
          </property>
          <property name="value">
           <number>350</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxShowParNo">
        <property name="text">
         <string>Show paragraph numbers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCharReplacement">
     <property name="toolTip">
      <string>Automatically replaces characters on input with other characters, e.g. the quote substitute character &quot; can be replaced with “”.</string>
     </property>
     <property name="title">
      <string>Character Replacement</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxAutoQuotes">
        <property name="toolTip">
         <string>Replaces the quote substitute character &quot;. To get a second-level quote, hold the Alt key and press the single-quote substitute character '.</string>
        </property>
        <property name="text">
         <string>AutoQuotes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxAutoQuotes">
        <item>
         <property name="text">
          <string notr="true">“…” / ‘…’</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">„…“ / ‚…‘</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string notr="true">»…« / ›…‹</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBoxAutoBrackets">
        <property name="toolTip">
         <string>Automatically inserts closing brackets for (, { and [ characters.</string>
        </property>
        <property name="text">
         <string>AutoBrackets</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxAutoApostrophe">
        <property name="toolTip">
         <string>Replaces the apostrophe substitute character ' with the appropriate unicode character ’.</string>
        </property>
        <property name="text">
         <string>AutoApostrophe</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBoxAutoDash">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
&lt;p&gt;Replaces multiple consecutive dashes:&lt;/p&gt;
&lt;dl&gt;
  &lt;dt&gt;2 dashes&lt;/dt&gt;
  &lt;dd&gt;en-dash (–)&lt;/dd&gt;
  &lt;dt&gt;3 dashes&lt;/dt&gt;
  &lt;dd&gt;em-dash (—)&lt;/dd&gt;
  &lt;dt&gt;4 dashes&lt;/dt&gt;
  &lt;dd&gt;two-em-dash (⸺)&lt;/dd&gt;
  &lt;dt&gt;5 dashes&lt;/dt&gt;
  &lt;dd&gt;three-em-dash (⸻)&lt;/dd&gt;
&lt;/dl&gt;
&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>AutoDash</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBoxAutoElipsis">
        <property name="toolTip">
         <string>Replaces three consecutive fullstops with the unicode elipsis character …</string>
        </property>
        <property name="text">
         <string>AutoElipsis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxLimitWidth</tabstop>
  <tabstop>spinBoxTextWidth</tabstop>
  <tabstop>checkBoxShowParNo</tabstop>
  <tabstop>checkBoxAutoQuotes</tabstop>
  <tabstop>comboBoxAutoQuotes</tabstop>
  <tabstop>checkBoxAutoBrackets</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxLimitWidth</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxTextWidth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
